File: 1878-check-if-array-is-sorted-and-rotated/check-if-array-is-sorted-and-rotated.java
class Solution {
    public boolean check(int[] nums) {
        int rotationpoint=0;
        for(int i=0;i<nums.length;i++){
            if(nums[i]>nums[(i+1)%nums.length]){
                rotationpoint++;
                if(rotationpoint>1){
                    return false;
                }
            }
        }
        return true;
    }
}

File: 1878-check-if-array-is-sorted-and-rotated/check-if-array-is-sorted-and-rotated.py
class Solution:
    def check(self, nums: List[int]) -> bool:
        rotationPoint=0

        for  i in range(len(nums)):
            if nums[i]>nums[(i+1)%len(nums)]:
                rotationPoint+=1
                continue
        
        if  rotationPoint>1:
            return False
        else:
            return True
            